.github/workflows/ci.yaml

```diff
@@ -1,4 +1,5 @@
-# When a third-party action is added (i.e., `uses`), please also add it to `download-licenses` in Makefile.
+# - When a third-party action is added (i.e., `uses`), please also add it to `download-licenses` in Makefile.
+# - The set of jobs in this file should be equal to that in ci-docs.yaml.
```

Comment:
Do we need to also doc in ci-docs.yaml that "The set of jobs in this file should be equal to that in ci.yaml."?

Reply:
Good point. Fixed in c379793. Also updated the comments in `ci.yaml` to be clearer when one needs to update `ci-docs.yaml`.